kapre/time_frequency.py

```diff
@@ -105,6 +105,7 @@ def __init__(
         pad_end=False,
         input_data_format='default',
         output_data_format='default',
+        use_parallel_stft=False,
```

Comment:
If we let users specify the number of processings, we'd need a variable for it here too. 
(This applies to all the places in `composed.py`, too)